packages/ses/src/cauterize-property.js

```diff
@@ -0,0 +1,69 @@
+import { objectHasOwnProperty } from './commons.js';
+
+/**
+ * @import {Reporter} from './reporting-types.js'
+ */
+
+/**
+ * Delete `obj[prop]` or at least make it harmless.
```

Comment:
I do not have the details paged into active brain right now, but this looks like it might be relevant to a mitigation @leotm needs for Hermes support.

Reply:
@leotm , care to comment? Thanks.

Reply:
https://github.com/endojs/endo/pull/2563 (and https://github.com/endojs/endo/pull/2334 ?), right?

Reply:
MetaMask folks reporting back after a travel/vacation/conference period. Sorry for missing the notifications on this.
This does look like it addresses the issue we worked around. We'll rebase and report back.

Reply:
sorry I missed this at Devcon then ill coming back (almost recovered)

just confirming this fixed _completePrototypes_ on Hermes after testing 🎉 thank you! closed https://github.com/endojs/endo/pull/2563 with the results

https://github.com/endojs/endo/pull/2334 still needed for CI feedback keeping SES compatible with Hermes
(after rebasing `master` five failing tests left before ready last for final re-review)